client/src/components/layouts/Booking/Booking.jsx

```diff
@@ -0,0 +1,166 @@
+import React, { useState, useEffect } from 'react';
+import InfiniteCalendar from 'react-infinite-calendar';
+import 'react-infinite-calendar/styles.css';
+import { Box, CircularProgress, Typography } from '@material-ui/core';
+import moment from 'moment';
+import axios from 'axios';
+import RadioButton from '../../commonComponents/TabsCheckout/FormFields/Radio';
+import exceptions from './exceptedDays';
+
+const { today, blockedWeeks, endDate, publicHolidays } = exceptions;
+const Booking = (props) => {
+  const [allReservations, setAllReservations] = useState(null);
+  const [TimeSlots, setTimeSlots] = useState(null);
+  const [availableTimeSlots, setAvailableTimeSlots] = useState(null);
+  const [isLoading, setIsLoading] = useState(true);
+  const fullReservedDays = [];
+  const storeTimeIds = {};
+  const storeAvailableTimeSlots = {};
+
+  const {
+    formField: { reservationTime },
+    covidAnswer,
+  } = props;
+
+  // get reservations and time slots data when the app mount
+  useEffect(() => {
+    // request to get all Reservations Data among 3 months
+    const requestOne = axios.post('/api/getReservationData', {
```

Comment:
I'd call these something like 'reservationRequest' and 'timeslotRequest' just to be super clear